src/Compilers/Core/Portable/Syntax/SeparatedSyntaxList.cs

```diff
@@ -598,7 +599,14 @@ IEnumerator IEnumerable.GetEnumerator()
             return new SeparatedSyntaxList<SyntaxNode>(nodes._list);
         }
 
-        public static implicit operator SeparatedSyntaxList<TNode>(SeparatedSyntaxList<SyntaxNode> nodes)
+        [Obsolete("Implicit downcast is not safe. Use explicit cast instead.", error: true)]
```

Comment:
Since this is no longer an operator. Consider to not refer to it as one. Something like: "This method is not safe and present for binary compatibility only. Use explicit cast instead."

Reply:
Now updated.